### PR TITLE
feat(docker-local): 1671 setup docker-compose to use thridparty-scheme-adapter 

### PR DESCRIPTION
**Pull request by eoln**

### CODE REVIEW

File: docker-local/docker-compose.yml [hunk 1]
```diff
@@ -336,14 +336,17 @@ services:
       - "dfspa-backend:172.17.0.1"
 
   dfspa-thirdparty-scheme-adapter-inbound:
-    image: "mojaloop/thirdparty-scheme-adapter:v11.15.0"
+    image: "mojaloop/thirdparty-scheme-adapter:v11.18.0"
+    # build:
```
Comment (kleyow) [inline docker-local/docker-compose.yml:340]: nit: these comments serve any purpose?

**Review by kleyow — APPROVED** [review]
lgtm

File: docker-local/docker-compose.yml (hunk 1 above)
Comment (eoln) [reply docker-local/docker-compose.yml:340]: I would like to keep these comments for a moment to have a shortcut for local building & testing

Comment (kleyow) [reply docker-local/docker-compose.yml:340]: :+1:

File: package.json [hunk 2]
```diff
@@ -28,12 +28,13 @@
...
     "test:bdd": "jest --testMatch '**/test/step-definitions/**/*.step.ts'",
-    "test:e2e": "jest --testMatch '**/test/e2e/**/*.(test|spec).ts'",
+    "test:e2e": "jest --runInBand --testMatch '**/test/e2e/**/*.(test|spec).ts'",
     "test:contract": "jest --testMatch '**/test/contract/**/*.(test|spec).ts'",
     "test:coverage": "jest --coverage --coverageThreshold='{}'",
     "test:coverage-check": "jest --coverage",
     "test:junit": "jest --reporters=default --reporters=jest-junit",
-    "test:unit": "jest --testMatch '**/test/unit/**/*.(test|spec).ts'"
+    "test:unit": "jest --testMatch '**/test/unit/**/*.(test|spec).ts'",
+    "docker:clean:all": "docker rm -f $(docker ps -a -q); docker volume rm (docker volume ls -q)"
```
Comment (lewisdaly) [inline package.json:37]: Why this as well as the shell script above?

**Review by lewisdaly — APPROVED** [review]

**Pull request merged**